main.go

```diff
@@ -10,6 +9,6 @@ import (
 func main() {
 	rootCmd := cmd.NewRootCommand()
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Fprintln(os.Stderr, err.Error())
+		os.Exit(1)
```

Comment:
Since you'll have to live with this. Do you want to somehow use a separate code for validation errors and other errors (i.e. can't load file?).

Reply:
Added separate exit codes for the different error paths

Reply:
Cute!